apps/www/registry/default/example/timeline-demo.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react"
+
+import {
+  Timeline,
+  TimelineContent,
+  TimelineDot,
+  TimelineHeading,
+  TimelineItem,
+  TimelineLine,
+} from "@/registry/default/ui/timeline"
+
+export default function TimelineDemo() {
+  return (
+    <Timeline>
+      <TimelineItem status="done">
+        <TimelineHeading>Plan!</TimelineHeading>
+        <TimelineDot status="done" />
+        <TimelineLine done />
+        <TimelineContent>
+          Before diving into coding, it is crucial to plan your software project
+          thoroughly. This involves defining the project scope, setting clear
+          objectives, and identifying the target audience. Additionally,
+          creating a timeline and allocating resources appropriately will
+          contribute to a successful development process.
+        </TimelineContent>
+      </TimelineItem>
+      <TimelineItem status="done">
+        <TimelineHeading side="right" className="text-destructive">
+          Design
+        </TimelineHeading>
+        <TimelineHeading side="right" className="text-destructive">
```

Comment:
This double TimelineHeading pattern seems really weird? Looks like it makes the font bold, but surely just adding font-bold to the classname is better?

Reply:
The TimelineHeading component does more than just make the font bold. You can check [here](https://github.com/flixlix/shadcn-ui/blob/618c4aa558fecde7df1f3b3a9e5011ec805f4aa9/apps/www/registry/default/ui/timeline.tsx#L138-L156) what it does exactly.

Basically it makes sure the heading is placed in the correct place. Since I am working with grid, depending on the side that is set as a prop, the correct columns will be used as limits for the heading. You can try setting this to a `p` component and see the chaos that emerges 😅.
This is also the reason you can set any `Timeline[[...]]` component in any order inside the TimelineItem component and it will still render in the correct position.

Reply:
Oh sure that makes sense, I just meant you see how on line 28 and 31 in that example demo, you have two TimelineHeadings that are doing the exact same thing? Or am I missing something there?

Reply:
oh yeah, you're right! 🤦
I didn't notice this at all